django/contrib/admin/options.py

```diff
@@ -324,6 +328,12 @@ def formfield_for_manytomany(self, db_field, request, **kwargs):
             form_field.help_text = (
                 format_lazy("{} {}", help_text, msg) if help_text else msg
             )
+        else:
+            msg = _("Select one element from the field.")
+            help_text = form_field.help_text
+            form_field.help_text = (
+                format_lazy("{} {}", help_text, msg) if help_text else msg
+            )
```

Comment:
Help message is not necessary in this case. Please remove this.

Reply:
Gotcha, I will get on that.

Reply:
Hey, I'm having some trouble with this unit test.

How do I explicitly pass widget.allow_multiple_selected into the options?

For the test you have to call it from the formfield_for_dbfield function which passes to the aforementioned options.py. However, it is not letting me pass allow_multiple_selected as a parameter.

Reply:
Nevermind, found it.